test/check-journal

```diff
@@ -208,5 +208,6 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
                      ])
 
         self.allow_restart_journal_messages()
+        self.allow_journal_messages(".*Failed to get realtime timestamp: Cannot assign requested address.*")
```

Comment:
This needs to go to the top of the test, since we will end the test early on line 177 on RHEL 7.